test/mix/tasks/phoenix.gen.channel_test.exs

```diff
@@ -76,4 +76,10 @@ defmodule Mix.Tasks.Phoenix.Gen.ChannelTest do
       Mix.Tasks.Phoenix.Gen.Channel.run ["Admin.Room", "rooms", "new_message"]
     end
   end
+
+  test "name can't already be defined" do
+    assert_raise Mix.Error, fn ->
+      Mix.Tasks.Phoenix.Gen.Channel.run ["Channel", "channels"]
```

Comment:
You will have to define dummy modules at the top of the file. I would call them "Dup". `Phoenix.DupChannel`, `Phoenix.DupHTMLView`, `Phoenix.DupHTMLController`, etc.